src/Libraries/Microsoft.AspNetCore.Diagnostics.Middleware/Logging/IncomingPathLoggingMode.cs

```diff
@@ -1,11 +1,17 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#if NET8_0_OR_GREATER
+
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.Shared.DiagnosticIds;
+
 namespace Microsoft.AspNetCore.Diagnostics.Logging;
 
 /// <summary>
 /// Strategy to decide how request path is logged.
 /// </summary>
+[Experimental(diagnosticId: Experiments.HttpLogging, UrlFormat = Experiments.UrlFormat)]
```

Comment:
Why add this?

Reply:
I marked all public API for this feature as experimental.

Reply:
Removed